draft-ietf-capport-architecture.xml

```diff
@@ -496,6 +496,189 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
+           </t>
+
+            <t>
+
+               The Enforcement Device needs to be aware of the physical interface,
+               which constrains the environment: it must either be part of the device providing
+               physical access (e.g.. implemented in firmware), or packets traversing the network
+               must be extended to include information about the source physical interface (e.g.
+               a tunnel).
+           </t>
+
+            <t>
+               The API server faces a similar problem, implying that it should co-exist with the
+               Enforcement Device, or that the enforcement device should extend requests to it
+               with the identifying information.
+            </t>
+           </section>
+           <section anchor="id_example_IP_address" title="IP Address">
+            <t>
+                A natural identifier to consider is the IP address of the User Equipment.
+                At any given time, no device on the network can have the same IP address
+                without causing the network to malfunction, so it is appropriate from the
+                perpsective of uniqueness.
+            </t>
+
+            <t>
+                However, it may be possible to spoof the IP address, particularly for
+                malicious reasons where proper funtioning of the network is not necessary
+                for the malicious actor. Consequently, any solution using the IP address
+                should proactively try to prevent spoofing of the IP address. Similarily,
+                if the mapping of IP address to User Equipment is changed, the components
+                of the architecture must remove or update their mapping to prevent spoofing.
+            </t>
+
+            <t>
+                Since the IP address may traverse multiple segments of the network, more
+                flexibility is afforded to the Enforcement Device and the API server: they
+                simply must exist on a segment of the network where the IP address is still
+                unique. However, consider that a NAT may be deployed between the User Equipment
+                and the Enforcement Device. In such cases, it is possible for the components
+                to still uniquely identify the device if they are aware of the port mapping.
+            </t>
+
+            <t>
+                In some situtations, the User Equipment may have multiple IP addresses, while
+                still satisfying all of the recommended properties. This raises some challenges
+                to the components of the network. For example, if the user equipment tries
+                to access the network with multiple IP addresses, should the enforcement device
+                and API server treat each IP address as a unique User Equipment, or should it
+                tie the multiple addresses together into one view of the subscriber?
+                An implementation MAY do either.
```

Comment:
This is the point where you talk about using the v6 prefix rather than the full /128.